client/src/extension.ts

```diff
@@ -34,7 +34,7 @@ const extensionObject = vscode.extensions.getExtension('flix.flix')
 
 export const FLIX_GLOB_PATTERN = '**/*.flix'
 
-const FPKG_GLOB_PATTERN = '**/lib/**/*.fpkg'
+export const FPKG_GLOB_PATTERN = '**/lib/**/*.fpkg'
```

Comment:
I notice now that this pattern looks strange.

I think this means that it takes any fpkg in a folder called lib. Is that what we want?

I would think we should only take from the root lib folder, i.e. `./lib/*.fpkg` or something like that. Can you test it?

Reply:
Yes! here the problem is with any pattern is that the pattern should match with the global path. So, `./lib/*.fpkg` won't match any of the `fkg` file.
For example, in my machine, the pattern should match with:
`/home/manoj/Desktop/summers/aarhus/testing/lib/testing.fpkg` 
whereas `./lib/testing.fpkg` will not be accepted here.

Instead of this, we can use `RelativePattern`. `vscode.workspace.workspaceFolders` gives the global path of the workspace directory, In our case, it returns an array of size 1 and we can update it like this:
`FPKG_GLOB_PATTERN = new vscode.RelativePattern(vscode.workspace.workspaceFolders?.[0], 'lib/**/*.fpkg')`

This works smoothly and only reads files inside the root `lib` folder. It also reads paths like `./lib/subLib/sample.fpkg`.